### ISSUE TITLE
A11y Build Warnings:  A11y: Non-interactive element <label> cannot have interactive role 'option'

### ISSUE THREAD

**Issue by cesarnml** [skeletonlabs/skeleton#1291]
### Current Behavior

SvelteKit: v1.11.0
Skeleton: v1.0.0

Several components are causing warnings to be emitted when building a project.

<img width="1680" alt="Screenshot 2023-04-07 at 10 05 42" src="https://user-images.githubusercontent.com/26180499/230532881-2c670603-61a7-46d3-b4c1-db0ecc0b6f0f.png">
![Screenshot 2023-04-07 at 10 05 42 (2)](https://user-images.githubusercontent.com/26180499/230532889-8c8ea44f-706f-4f72-9acf-64689be9e1a2.png)

### Steps To Reproduce

1. Run `vite build`

### Anything else?

Encountered warning while building portfolio site (link below).  Started happening after skeleton v1.0 upgrade.

https://github.com/cesarnml/portfolio-sveltekit

**Comment by endigo9740** [comment]
@cesarnml already resolved and the fix will come in the new release next week. Thanks for reporting though!

https://github.com/skeletonlabs/skeleton/pull/1280#issuecomment-1497926290
